packages/trpc/server/routers/viewer/bookings.tsx

```diff
@@ -672,6 +672,8 @@ export const bookingsRouter = router({
           recurringEvent: parseRecurringEvent(booking.eventType?.recurringEvent),
           location,
           destinationCalendar: booking?.destinationCalendar || booking?.user?.destinationCalendar,
+          seatsPerTimeSlot: booking.eventType?.seatsPerTimeSlot,
+          seatsShowAttendees: booking.eventType?.seatsShowAttendees,
```

Comment:
When changing location, we pass in a `CalendarEvent`. It was missing seats data which would trigger hiding other attendees from each other on seats events.